seml/database.py

```diff
@@ -277,3 +281,40 @@ def clean_unreferenced_artifacts(db_collection_name=None, yes=False):
     logging.info('Deleting not referenced artifacts...')
     delete_files(db, not_referenced_artifacts, progress=True)
     logging.info(f'Successfully deleted {n_delete} not referenced artifact{s_if(n_delete)}.')
+
+def list_database(pattern, mongodb_config=None, progress=False, list_empty=False):
+    """
+    Prints a tabular version of multiple collections and their states (without resolving RUNNING experiments that may have been canceled manually).
+
+    Parameters
+    ----------
+    pattern : str
+        The regex collection names have to match against
+    mongodb_config : dict or None
+        A configuration for the mongodb. If None, the standard config is used.
+    progress : bool
+        Whether to use a progress bar for fetching
+    list_empty : bool
+        Whether to list collections that have no documents associated with any state
+    """
+    logging.warning(f"Status of {States.RUNNING[0]} experiments may not reflect if they have died or been canceled. Use `seml ... status` instead.")
+    if mongodb_config is None:
+        mongodb_config = get_mongodb_config()
+    db = get_database(**mongodb_config)
+    collection_names = [name for name in db.list_collection_names()
+                        if name not in ('fs.chunks', 'fs.files') and re.compile(pattern).match(name)]
+    name_to_counts = {}
+    it = tqdm(collection_names) if progress else collection_names
+    
+    for collection_name in it:
+        counts_by_status = db[collection_name].aggregate([{'$group' : {'_id' : '$status', '_count' : {'$sum' : 1}}}])
+        name_to_counts[collection_name] = Counter({result['_id'] : result['_count'] for result in counts_by_status})
+    
+    columns = [States.STAGED, States.PENDING, States.RUNNING, States.FAILED, States.KILLED, States.INTERRUPTED, 
```

Comment:
wouldn't this just be `States.keys()`?

Reply:
Since python 3.6 insertion order of dicts is preserved and this is touted as a feature not a mere implementation. Thus, we can just take the same order as in settings.py with less redundancy on the ordering.